externals/llvm-external-projects/torch-mlir-dialects/test/Dialect/Tcp/misc.mlir

```diff
@@ -62,3 +62,48 @@ func.func @test_broadcast_axes_w_duplicates(%arg0 : tensor<?x1x?x1xf32>, %arg1 :
   %0 = "tcp.broadcast"(%arg0, %arg1, %arg2) {axes = [1, 1]} : (tensor<?x1x?x1xf32>, index, index) -> tensor<?x?x?x?xf32>
   return %0 : tensor<?x?x?x?xf32>
 }
+
+// -----
+
+// CHECK-LABEL: func.func @test_group(
```

Comment:
Do we need tests that check the verifier fails as intended (e.g. when there is no `yield` op)?

Reply:
Typically, I don't add tests for the cases that are controlled by the traits, which is the case for the presence of a `yield` op. But I realized that the traits don't verify other constraints: the number and type of `yield` args matching the result.

Add verifier to check for all of those and also added appropriate tests.